codes/c/chapter_dynamic_programming/min_path_sum.c

```diff
@@ -0,0 +1,125 @@
+/**
+ * File: min_path_sum.c
+ * Created Time: 2023-10-02
+ * Author: Zuoxun (845242523@qq.com)
+ */
+
+#include "../utils/common.h"
+
+int n, m;
+int grid_size, grid0_size;
+
+/* 最小路径和：暴力搜索 */
+int minPathSumDFS(int grid[][m], int i, int j) {
```

Comment:
The formal parameter passing of this function is too awkward. Global variables are included in the parameter passing process, which is very inflexible.

Can be used as follows：

```c
int minPathSumDFS(int gridCols, int grid[][gridCols], int i, int j) {
```

Reply:
Thank you, learned a lot, i've edited it following your suggestion.